src/server/schema/resolvers/queries/getLife.ts

```diff
@@ -0,0 +1,15 @@
+import { ObjectId } from "mongodb";
+import { GraphQLQueryResolvers } from '../definitions';
+import { getDatabaseContext } from '../../../database';
+
+const query: GraphQLQueryResolvers['getLife'] = async (
+    root,
+    { id }
+) => {
+    const { collections } = await getDatabaseContext();
+
+    const result = await collections.lives.findOne({ _id: new ObjectId(id) });
+    return result;
```

Comment:
`id` should be of GraphQL type `ObjectID` which is mapped to `ObjectId`, also you may directly return the promise